moveit_core/collision_detection_bullet/src/collision_env_bullet.cpp

```diff
@@ -250,8 +250,8 @@ void CollisionEnvBullet::addToManager(const World::Object* obj)
   }
 
   collision_detection_bullet::CollisionObjectWrapperPtr cow(new collision_detection_bullet::CollisionObjectWrapper(
-      obj->id_, collision_detection::BodyType::WORLD_OBJECT, obj->shapes_, obj->shape_poses_, collision_object_types,
-      false));
+      obj->id_, collision_detection::BodyType::WORLD_OBJECT, obj->shapes_,
+      getWorld()->getGlobalShapeTransforms(obj->id_), collision_object_types, false));
```

Comment:
Why not directly accessing `obj->global_shape_poses_`?
```suggestion
      obj->id_, collision_detection::BodyType::WORLD_OBJECT, obj->shapes_, obj->global_shape_poses_,
      collision_object_types, false));
```